Makefile

```diff
@@ -0,0 +1,6 @@
+test:
+	tox
+clean:
```

Comment:
minor suggestion to change this to a find stanza

find -name *.pyc -delete 

tomatoe/tomatoe